### PR TITLE
Zhenya/accounts

### PR DESCRIPTION
**Pull request by zhenyasav**
This makes cases when popup blockers block auth dialogs a little neater w.r.t. what gets thrown. Also tweaked the height of a popup window when `Meteor.loginWithMeteorDeveloperAccount({loginStyle: 'popup'})` to help fit the UI without scrollbars.

### CODE REVIEW

**Comment by debergalis** [comment]
thumbs up

File: packages/meteor-developer/meteor_developer_client.js [hunk 1]
```diff
@@ -40,7 +40,7 @@ var requestCredential = function (options, credentialRequestCompleteCallback) {
     loginUrl: loginUrl,
     credentialRequestCompleteCallback: credentialRequestCompleteCallback,
     credentialToken: credentialToken,
-    popupOptions: {width: 470, height: 420}
+    popupOptions: {width: 470, height: 490}
```
Comment (robertpitt) [inline packages/meteor-developer/meteor_developer_client.js]: Can we not make these configurable within the `ServiceConfiguration`, we have our own Oauth 2 implementation for our customers and are about to switch to dialog based authentication within the next sprint.

When I start work in this I can do a PR for the changes if a team member gives it the thumbs up.

**Comment by dgreensp** [comment]
Is this something user-visible, i.e. does the error get caught somewhere and shown to the user?

**Comment by zhenyasav** [comment]
This isn't user visible (if you're referring to the new `throw` I put in) it's inside the `Meteor.loginWith<Service>` flow, which assumes nothing about UI. Is that the answer you were looking for?

**Comment by dgreensp** [comment]
Yeah, sounds good!  Thanks.  Merge it.

**Pull request merged**